a2.js

```diff
@@ -0,0 +1,37 @@
+document.addEventListener('DOMContentLoaded', ()=>{
+    let select = document.querySelector("#select")
```

Comment:
Here you are query selecting an `id` that is called `select`, but if you check your `<select>` tag you don't give an id or class to it. You could have querySelector('select')